allennlp/__main__.py

```diff
@@ -0,0 +1,26 @@
+from typing import Sequence
+import argparse
+import logging
+import sys
+
+from allennlp.commands.bulk import add_bulk_subparser
+from allennlp.common.params import PARAMETER
+
+# disable parameter logging
+logging.disable(PARAMETER)
```

Comment:
yeah, I added a TODO to figure this out, the timing is sort of tricky, I'll figure it out when I switchover the train_model